pkg/gui/commits_panel.go

```diff
@@ -64,12 +64,24 @@ func (gui *Gui) handleCommitSelect() error {
 func (gui *Gui) refreshReflogCommitsConsideringStartup() {
 	switch gui.State.StartupStage {
 	case INITIAL:
+		var wg sync.WaitGroup
+		wg.Add(1)
+
 		go utils.Safe(func() {
 			_ = gui.refreshReflogCommits()
 			gui.refreshBranches()
 			gui.State.StartupStage = COMPLETE
+			wg.Done()
 		})
+		go utils.Safe(func() {
+			// The github cli can be quite slow so we load the github PRs sparately
+			if gui.Config.GetUserConfig().Git.EnableGhCommand {
+				gui.refreshGithubPullRequests()
```

Comment:
I suspect that by having `refreshGithubPullRequests` only called from here, we won't be refreshing the PRs periodically, but we should do that. Let me know if you need pointers for that.

Reply:
I had left the logic of the original pr implementation in place.
I think we would like to update it when the PR status is updated,
but in that case I did not know where it would be appropriate to call this function.